packages/components/table/src/table-body/styles-helper.ts

```diff
@@ -29,11 +29,10 @@ function useStyles<T>(props: Partial<TableBodyProps<T>>) {
     const classes = [ns.e('row')]
     if (
       parent?.props.highlightCurrentRow &&
-      row === props.store.states.currentRow.value
+      ref(row).value === props.store.states.currentRow.value
```

Comment:
```js
row === unref(props.store.states.currentRow)
```
WDYT?

Reply:
What is the point of `ref(row).value` here?